tests/acme_diags/derivations/test_acme.py

```diff
@@ -89,17 +89,53 @@ def setUp(self):
         self.mock_file_axis: "FileAxis" = Mock()
         self.mock_file_axis.getData.return_value = np.arange(0, 1002)
 
-    def test_low_none(self):
-        actual = determine_tau(self.mock_file_axis, None, 10)
-        expected = "tau <10"
-        self.assertEqual(actual, expected)
-
-    def test_high_none(self):
-        actual = determine_tau(self.mock_file_axis, 5, None)
-        expected = "5< tau"
-        self.assertEqual(actual, expected)
-
-    def test_low_high(self):
-        actual = determine_tau(self.mock_file_axis, 5, 10)
-        expected = "5< tau < 10"
-        self.assertEqual(actual, expected)
+        # Need to mock __getitem__ in order to provide list value
+        self.mock_file_axis.__getitem__ = Mock()
+        self.mock_file_axis.__getitem__.side_effect = [0, 10]
+
+    def test_low_arg_only(self):
```

Comment:
After fixing the issue by mocking `self.mock_file_axis.__getitem__`, the tests themselves were breaking.

`determine_tau` returns a tuple of variables, not a single `str` so these assertion weren't correct.